pm/sigverifier.go

```diff
@@ -31,39 +28,39 @@ func (sv *DefaultSigVerifier) Verify(addr ethcommon.Address, msg, sig []byte) bo
 // mapping ETH addresses to approved signer sets. This implementation will
 // recover a ETH address from a signature and check if the recovered address
 // is approved
-type ApprovedSigVerifier struct {
-	broker Broker
-}
+// type ApprovedSigVerifier struct {
```

Comment:
Approved signers is temporarily disabled since the signer registry was moved out of the TicketBroker contract. Commenting out this code for now and we can reintroduce it when we re-enable approved signers at a contract level